pkg/controller/controller_binding_test.go

```diff
@@ -1809,3 +1993,51 @@ func TestReconcileBindingWithSecretConflictFailedAfterFinalRetry(t *testing.T) {
 		}
 	}
 }
+
+// TestReconcileServiceInstanceCrednetialWithStatusUpdateError verifies that the
+// reconciler returns an error when there is a conflict updating the status of
+// the resource
+func TestReconcileServiceInstanceCredentialWithStatusUpdateError(t *testing.T) {
+	fakeKubeClient, fakeCatalogClient, _, testController, sharedInformers := newTestController(t, fakeosb.FakeClientConfiguration{
+		BindReaction: &fakeosb.BindReaction{
```

Comment:
I don't think we need this because Bind should not have been called.  Also, looks like this test is missing assertions on the various clients.

Reply:
You are correct. Now that there is a Status update that happens prior to sending the request, the bind request is no longer made.

Reply:
You are correct. Now that there is a Status update that happens prior to sending the request, the bind request is no longer made.